java-frontend/src/test/java/org/sonar/java/se/CheckerDispatcherTest.java

```diff
@@ -49,7 +49,7 @@ private static ExplodedGraphWalker mockExplodedGraphWalker() {
     ExplodedGraphWalker explodedGraphWalker = mock(ExplodedGraphWalker.class);
     explodedGraphWalker.programPosition = new ProgramPoint(new CFG.Block(1));
     explodedGraphWalker.programState = mock(ProgramState.class);
-    explodedGraphWalker.node = new ExplodedGraph.Node(explodedGraphWalker.programPosition, explodedGraphWalker.programState);
+    explodedGraphWalker.node = new ExplodedGraph().node(explodedGraphWalker.programPosition, explodedGraphWalker.programState);
```

Comment:
I disagree, this change is related to the fact that Node constructor is now private.